racetrack_client/racetrack_client/utils/quantity.py

```diff
@@ -81,3 +81,19 @@ def __truediv__(self, other) -> 'Quantity':
     def plain_number(self) -> float:
         """Convert quantity to a plain number without any suffixes"""
         return self.base_number * self._suffix_multipliers[self.suffix]
+
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, v):
+        if v is None:
+            return None
+        return Quantity(str(v))
+
+    @classmethod
+    def __modify_schema__(cls, field_schema):
+        field_schema.update(
+            examples=["128974848", "129M", "128974848000m", "123Mi", "0.129G"],
+        )
```

Comment:
Hmm, are those three functions needed only to get better Pydantic compatibility? And they're the only methods added for Pydantic? If so, I propose to mark them as such with comment, so coders will understand where to look for more documentation (or maybe only I don't know Pydantic).

Reply:
Yes, without these lines, new Pydantic fails to operate. And yes, I'll add comments that it's intended for pydantic.